sample/build.gradle

```diff
@@ -148,6 +150,24 @@ if (isCi) {
           }
         }
   }
+
+  tasks.withType(R8Task)
+        .matching { it.name == "minifyExternalStagingWithR8" }
+        .configureEach { r8Task ->
+          doLast {
+            println "Checking expected configuration contains embedded library rules from androidTest"
+            def output = r8Task.getProguardConfigurationOutput()
+            String allConfigurations
+            if (output instanceof Provider<File>) {
+              // This is a Provider on AGP 7
+              allConfigurations = output.get().text
+            } else {
+              allConfigurations = output.text
+            }
+            PreconditionsKt.check(allConfigurations.contains("-keep class slack.test.only.Android { *; }"))
```

Comment:
Is this something we could move into a test instead? (I guess reaching into task outputs isn't super easy?)

Reply:
right, the sample acts as an integration test too